lnwallet/channel.go

```diff
@@ -109,7 +109,7 @@ type channelState uint8
 const (
 	// channelPending indicates this channel is still going through the
 	// funding workflow, and isn't yet open.
-	channelPending channelState = iota
+	channelPending channelState = iota // nolint: unused
```

Comment:
I don't think this enum is actually serving a purpose at all, perhaps we can remove it completely?

Reply:
What do you mean exactly? `channelState` is used. This particular value not, but I didn't want to remove it to change the numbering (serialization in mind perhaps)

Reply:
I don't think `LightningChannel` is serialized to disk. I was referring more to if you look at the uses of this enum (not just `channelPending`) within `LightningChannel`, it doesn't seem to be doing much.

Reply:
I gave it a shot, but the closed state is still set and checked. I'd say, "not in this pr" ?

Reply:
Sure can be left for a follow-up. I think it's just outdated code.